change/@azure-msal-browser-e3d4b10b-bba6-47de-9ff2-7ee4039bb97e.json

```
@@ -0,0 +1,7 @@
+{
+    "type": "patch",
```

I think this PR qualifies as "minor" since it adds a new feature.

```suggestion
    "type": "minor",
```